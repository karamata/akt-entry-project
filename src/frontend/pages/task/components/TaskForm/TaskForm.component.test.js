import React from 'react';
import renderer from 'react-test-renderer';

import TaskFormComponent from './TaskForm.component';

jest.mock('antd/dist/antd.css', () => <div />);

it('Create form submit a task', () => {
  const component = renderer.create(<TaskFormComponent />);
  expect(component).toMatchSnapshot();
});
