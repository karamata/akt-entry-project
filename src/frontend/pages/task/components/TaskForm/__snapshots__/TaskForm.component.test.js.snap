// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create form submit a task 1`] = `
<span
  className="ant-input-affix-wrapper"
  style={
    Object {
      "margin": "15px 0",
    }
  }
>
  <input
    className="ant-input"
    name="title"
    onChange={[Function]}
    onKeyDown={[Function]}
    style={null}
    type="text"
    value=""
  />
  <span
    className="ant-input-suffix"
  >
    <button
      className="ant-btn ant-btn-primary"
      onClick={[Function]}
      type="button"
    >
      <span>
        Add
      </span>
    </button>
  </span>
</span>
`;
